doc/extensibility/dataflow_example.md

```diff
@@ -0,0 +1,168 @@
+# DataFlow Examples
+
+## Original Data Source
+
+``` Csharp
+/// <summary>
+/// An example of an original datasource.
+/// The goal is to listen to an original source and publish a new value whenever it changes.
+/// </summary>
+[Export]
```

Comment:
Typically Export(typeof(IDataFlowContract)), instead of the private class itself...?